unstable/wlr-output-manager-v1.xml

```diff
@@ -0,0 +1,350 @@
+        will not be the same if the compositor does not scale the surface
+        buffers.
+
+        Combined with the current_mode and transformation event, this event
+        effectively describes the scale applied to the output's buffer
+        before it is displayed.
+
+        This event is sent after the client binds to the output_device object
+        and whenever the size of the output device in the global compositor
+        space changes.
+      </description>
+      <arg name="width" type="int"
+        summary="width in global compositor space"/>
+      <arg name="height" type="int"
+        summary="height in global compositor space"/>
+    </event>
+
+    <event name="position">
+      <description summary="position of the output in the compositor space">
+        This events describes the position of the output_device within the
+        global compositor space.
+
+        If the compositor implements xdg_output and this output is enabled,
+        the zxdg_output_v1.logical_position must report the same position. 
+
+        This event is sent after the client binds to the output_device
+        and whenever the position of the output changes in the compositor space.
+      </description>
+      <arg name="x" type="int"
+        summary="x position within the global compositor space"/>
+      <arg name="y" type="int"
+        summary="y position within the global compositor space"/>
+    </event>
+    
+    <enum name="enablement">
+      <description summary="describes enabled state">
+        Describes whether a device is enabled, i.e. device is used to
+        display content by the compositor. This wraps a boolean around
+        an int to avoid a boolean trap.
+      </description>
+      <entry name="disabled" value="0"/>
+      <entry name="enabled" value="1"/>
+    </enum>
+
+    <event name="enabled">
+      <description summary="output is enabled or disabled">
+        This event describes whether the output is actively displaying a part
+        of the global compositor space.
+
+        If an output_device is enabled, the output must be represented by
+        a wl_output.
+        
+        This event is sent after the client binds to the output_device
+        and whenever the output device is enabled or disabled.
+      </description>
+      <arg name="enabled" type="int" summary="output enabled state"/>
+    </event>
+
+    <event name="name">
+      <description summary="name of this output">
+        Many compositors will assign names to their outputs, show them to the
+        user, allow them to be configured by name, etc. The client may wish to
+        know this name as well to offer the user similar behaviors.
+
+        The naming convention is compositor defined, but limited to
+        alphanumeric characters and dashes (-). Each name is unique among all
+        wl_output globals, but if a wl_output global is destroyed the same name
```

Comment:
`wl_output` globals? :)

I think you copy+pasted a bit eagerly.

Should probably reference the `xdg-output` name event (and force equality?).